pubspec.yaml

```diff
@@ -1,18 +1,18 @@
 name: test_process
-version: 1.0.6-dev
+version: 2.0.0-nullsafety.0
 
 description: A package for testing subprocesses.
 homepage: https://github.com/dart-lang/test_process
 
 environment:
-  sdk: '>=2.1.0 <3.0.0'
+  sdk: ">=2.12.0-0 <3.0.0"
 
 dependencies:
-  async: ^2.0.0
-  meta: ^1.0.0
-  path: ^1.0.0
-  test: ^1.0.0
+  async: ^2.5.0-nullsafety.0
```

Comment:
many of these are stable now. Drop the `-nullsafety` bits where you can!

Reply:
Yep, I updated all of them.